### DISCUSSION TITLE
Toast: provide default close callback to headless toasts

### DISCUSSION THREAD

**Discussion by Cr3aHal0** [primefaces/community#740] [PrimeNG Templates] [answered]
Hey guys,

Headless toast is a wonderful addition to PrimeNG 17, don't get me wrong. What's strange is that you can customize the content but still not provide a clean way to close an headless toast without clearing the service toast "chanel". Could default toast close callback be provided to headless template so that it can be closed like any other toast ?

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Thanks for the feedback, we've added it to our internal backlog.

**Reply by Cr3aHal0** [reply]
Hi ! Thanks a lot for the reply. Just in case, I did provide a PR for this feat : https://github.com/primefaces/primeng/pull/14418
